Content/posts/penny-update

```diff
@@ -0,0 +1,33 @@
+---
+date: 2023-05-16 
+description: Penny Update - Keep Up With Swift's Evolution
+tags: penny, bot, discord
+author: Mahdi
+authorImageURL: /author-images/mahdi.jpg
+---
+# Penny
```

Comment:
Let's add a more descriptive title - this is used on the homepage, in the open graph data etc